src/project/manifest/system_requirements.rs

```diff
@@ -27,9 +27,6 @@ pub struct SystemRequirements {
 
     /// Dictates information about the libc version (and optional family).
     pub libc: Option<LibCSystemRequirement>,
-
-    /// Information about the system architecture.
-    pub archspec: Option<String>,
```

Comment:
Removing this is a breaking change, I would like to move this into a (deprecation) warning if used.